frontend/jest.config.cjs

```diff
@@ -6,6 +6,7 @@ module.exports = {
     "^.+\\.(css|less|scss)$": "babel-jest",
   },
   coverageReporters: ["text", "cobertura", "lcov"],
+  testPathIgnorePatterns: ["<rootDir>/node_modules/", "<rootDir>/e2e/"],
```

Comment:
Jest was picking up the playwright tests which also include `.spec`, causing Jest to fail in CI. This excludes the e2e directory to avoid the problem.